tools/make/image.mk

```diff
@@ -48,7 +48,7 @@ image.build.%: go.build.%
 	@echo "===========> Building image $(IMAGES) in tag $(TAG) for $(IMAGE_PLAT)"
 	$(eval BUILD_SUFFIX := --pull -t $(IMAGE):$(TAG) -f $(ROOT_DIR)/tools/docker/$(IMAGES)/Dockerfile bin)
 	@echo "===========> Creating image tag $(REGISTRY)/$(IMAGES):$(TAG) for $(ARCH)"; \
-	$(DOCKER) build --platform $(IMAGE_PLAT) $(BUILD_SUFFIX)
+	$(DOCKER) build --build-arg TARGETPLATFORM=$(IMAGE_PLAT) $(BUILD_SUFFIX)
```

Comment:
can you try replacing `$DOCKER` with `DOCKER_BUILDKIT=1 docker` ?

Reply:
Cool, that works, let me update it.